backend/app/services/chat/ws_emitter.py

```
@@ -949,9 +954,24 @@ async def safe_emit_in_main_loop(
     if _main_event_loop and _main_event_loop.is_running():
-        asyncio.run_coroutine_threadsafe(emit_func(*args, **kwargs), _main_event_loop)
+        future = asyncio.run_coroutine_threadsafe(
+            emit_func(*args, **kwargs), _main_event_loop
+        )
+
+        # Optionally add a callback to log errors (but don't wait for result)
+        # This is purely for debugging purposes
+        def _log_error(f: Any) -> None:
+            try:
+                f.result()
+            except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)